tests/2016-10-01/Microsoft.KeyVault.tests.json

```diff
@@ -0,0 +1,121 @@
+{
+    "tests": [
```

Comment:
>     [](start = 0, length = 4)

please format this file and use consistent indents. we use 2 spaces across this repo.